book/website/collaboration/leadership/leadership-building.md

```diff
@@ -0,0 +1,103 @@
+Therefore, leaders should understand how their positions affect the dynamics when working with others in their team, even when not exerting their power.
+Understanding power dynamics helps leaders identify and resolve conflict, build consensus, and create a more effective team culture.
+- **Acknowledging fairly**: Leadership roles come with a lot of visibility for their roles.
+This is partly earned by leaders' work as individuals but is often enhanced by the work of others in their teams. 
+Open leaders recognise and celebrate all kinds of work, appreciate and celebrate them openly and give appropriate credit to everyone, including those whose work makes them visible often taking invisible responsibilities for the sake of the overall success.
+
+##  What should Leaders Avoid
+
+- **Elitism and hypocrisy**: Open leadership in research in general benefits a lot from equity, inclusivity and diversity (EDI). 
+Thus, traits that sabotage EDI goals, should be avoided at all costs.
+Attitudes like elitism and hypocrisy can have unintended consequences on the project as well as on the leaders themselves.
+Elitism is when leaders constantly reinforce their authority, status, and expertise, creating distance between them and junior/new members of their teams.
+Hypocrisy is holding others accountable for one set of quality measures, and having a lesser standard for themselves.
+Leaders should hold the same standards for everyone and create spaces for collaboration through egalitarian rather than authoritative means.
+- **Ignoring the personal side of the team**: members of the team/organisation can disclose or not as much as they want from their personal lives. 
+But it is up to the leader to ensure that their team members feel safe to express the issues that affect their work such as their physical or mental health, and understand each others' challenges such as those caused by invisible or visible disability or caring responsibilities outside work and can seek help from each other or professionals.
+If leaders don't make it ok for their team members to share their sides, team members most likely (and even newer members of the team), would not know what is appropriate and won't ask for help even when they need it. 
+This will likely have an impact on the overall project/team/organisation dynamics, opportunities for building trustworthy teams and often their mental health (which may result in burnout).
+- **Tunnel vision**: Leaders should be aware of the research landscape, trends and state-of-the-art technology/approaches relevant to their teams. 
+Even if the organisation or a project might focus on one line of work, awareness about and an understanding of what is 'going on around' at the broader level will prompt further discussions and go beyond finding quick solutions.
```

Comment:
I don't understand the second part of what this sentence is trying to say - are we saying that a broader awareness might help with novel and innovative connections and support creative problem solving? Or something else?

Reply:
```suggestion
Even if the organisation or a project might focus on one line of work, awareness about and an understanding of what is 'going on around' at the broader level help create a better direction for the project.
They should prompt their team members to engage in discussions that help them go beyond finding quick solutions.
```